src/views/v2/components/events-bar/search-button/icon.php

```diff
@@ -10,7 +10,6 @@
  * @link http://m.tri.be/1aiy
  *
  * @version 4.9.10
- *
  */
-?>
-<span class="tribe-events-c-events-bar__search-button-icon tribe-common-svgicon"></span>
+
+$this->template( 'components/icons/search', [ 'classes' => [ 'tribe-events-c-events-bar__search-button-icon-svg' ] ] );
```

Comment:
if all we're doing is replacing this with another template call, we might as well just replace the template call to the search button icon itself.